rudder-webapp/SOURCES/rudder-webapp-postinst

```diff
@@ -168,6 +162,13 @@ echo "Done"
 # Adjust permissions on /var/rudder/configuration-repository
 /opt/rudder/bin/rudder-fix-repository-permissions  >> ${LOG_FILE}
 
+# Restore the plugins and checking them
```

Comment:
I dont get this. Currently we restore the status, then rudder-upgrade runs `check-compatibility` and `rudder-postupgrade`. Why doing it again just after?

Reply:
parent ticket change the order, but it wasn't such a great idea